libr/anal/sign.c

```diff
@@ -958,14 +958,8 @@ R_API int r_sign_all_functions(RAnal *a, bool merge) {
 		RSignItem *it = NULL;
 		if (merge || !name_exists (a->sdb_zigns, realname, sp)) {
 			it = item_from_func (a, fcn, realname);
-		} else {
```

Comment:
the fix looks good and i think thats a good default behaviour, but maybe we should allow users to have the same signature twice too. maybe we can have the `zign.dups` config variable for this, and maybe we need to break the abi to add such option inside RAnalOptions. but i think is better to have this feature as an option and disabled by default rather than removing the code. what do you think?